tests/unit/resizable/core.js

```diff
@@ -244,4 +244,57 @@ QUnit.test( "nested resizable", function( assert ) {
 	outer.remove();
 } );
 
+QUnit.test( "Resizable with scrollbars and box-sizing: border-box", function( assert ) {
+	assert.expect( 2 );
+
+	var elementContent = $( "<div>" )
+			.css( {
+				width: 50,
+				height: 200,
+				padding: 10,
+				border: 5,
+				borderStyle: "solid"
+			} )
+			.appendTo( "#resizable1" ),
+	element = $( "#resizable1" ).css( { overflow: "auto" } ).resizable(),
+	handle = ".ui-resizable-se";
+
+	$( "*" ).css( "box-sizing", "border-box" );
```

Comment:
That's unnecessarily slow & not clean - every element is selected and has an inline style added to set `border-box`. If you want a test with global `border-box`, add it via a `<style>` tag.

The same applies to other places.

Reply:
Ok, I'll change it!

Would it be better if I append a `<style>` tag using jQuery and remove it after the tests?
Something like this:
```javascript
var style = $( "<style> * { box-sizing: border-box; } </style>" ).appendTo( "head" );
//do test
style.remove();
``` 

One more thing. I noticed that other tests (not my new ones) do the same (see https://github.com/jquery/jquery-ui/blob/main/tests/unit/resizable/options.js#L558 ). Should I change those as well?

Thanks!

Reply:
You can also change that test. If there are too many of them, though, you don't have to do this here.